Resources/Prototypes/Research/industrial.yml

```
@@ -183,6 +183,7 @@
     - OreBagOfHolding
     - MiningDrillDiamond
     - AdvancedMineralScannerEmpty
+    - SalvageExpeditionsComputerCircuitboard # Corvax-Cringe
```

```suggestion
    - SalvageExpeditionsComputerCircuitboard
```
Yeah I am corvax enjoyer, how could you tell?

---

```suggestion
    - SalvageExpeditionsComputerCircuitboard # Corvax
```